cookbook/agent_concepts/knowledge/vector_dbs/couchbase_db.py

```diff
@@ -0,0 +1,128 @@
+# install couchbase-sdk - `pip install couchbase`
```

Comment:
```suggestion
"""
This example shows how to use the Couchbase vector database.

First install couchbase-sdk: `pip install couchbase`

For a local docker setup, you can use the following command to start the couchbase server:

```bash
./cookbook/scripts/run_couchbase.sh
```

Then go to the couchbase web ui at http://localhost:8091 and create a cluster with the following settings:

- Name: agno-cluster
- Username: admin
- Password: password

You can select "Finish with defaults". 

Next export the following environment variables:

```bash
export COUCHBASE_CONNECTION_STRING="couchbase://localhost"
export COUCHBASE_USER="admin"
export COUCHBASE_PASSWORD="password"
```
"""
```